### ISSUE TITLE
ROS2 xml launch files

### ISSUE THREAD

**Issue by Rayman** [ctu-mrs/ros2_uav_example#2]
Launch files can be written in Python, but also in xml, see also this tutorial: https://index.ros.org/doc/ros2/Tutorials/Launch-files-migration-guide/

The syntax changed though :(

**Comment by klaxalk** [comment]
Yes, however, I have not found how to run Components from the new XML format. And since they say Python is the preferred way in ROS2, I have not invested too much time into the XML variant.
